app/assets/javascripts/govuk_publishing_components/components/layout-super-navigation.js

```diff
@@ -0,0 +1,70 @@
+window.GOVUK = window.GOVUK || {}
+window.GOVUK.Modules = window.GOVUK.Modules || {};
+
+(function (Modules) {
+  function SuperNavigationToggle ($module) {
+    this.$module = $module
+    this.$button = _setupButton($module)
```

Comment:
I used it because there's a generic informal convention in the wide JavaScript community that an underscore indicates a private function - but appreciate that this isn't documented anywhere so is quite confusing. It's been updated to be `this.setupButton($module)`:

https://github.com/alphagov/govuk_publishing_components/blob/951b14bb286ae910bb3104b2ce61dd017f910eae/app/assets/javascripts/govuk_publishing_components/components/layout-super-navigation-header.js#L21-L37

Reply:
Would be great to open this discussion internally as well and we may be able to adopt it, indeed.